packages/TelegramClient-UI.package/TCUChatListItem.class/instance/lastUserName.st

```diff
@@ -0,0 +1,9 @@
+accessing
+lastUserName
+
+	^ ((self chat lastMessageSenderId = self chat core loggedInUserId) ifTrue:[
+		'You: ']
+		ifFalse: [
+			(self chat lastMessageSenderName) ifNil: [
```

Comment:
lastMessageSenderName is always initialized with empty string, we quickly changed it to ifEmpty